src/xo-app/vm/tab-general.js

```diff
@@ -23,7 +25,14 @@ import {
   XvdSparkLines
 } from 'xo-sparklines'
 
-export default ({
+export default connectStore(() => {
+  const getLastShutdownTime = createGetLastShutdownTime()
+  return (state, props) => {
+    return { lastShutdownTime: getLastShutdownTime(state, props.vm) }
+  }
+})(
```

Comment:
Instead of returning a selector to an object, you can return an object of selectors:

```js
return {
  lastShutdownTime: getLastShutdownTime
}
```